README.md

```diff
@@ -80,6 +80,12 @@ SecretKeyFactory:
 * HkdfWithHmacSHA256
 * HkdfWithHmacSHA384
 * HkdfWithHmacSHA512
+* ConcatenationKdfWithSHA224 (not available in FIPS builds)
```

Comment:
Double check that we need SHA224 support. It very weak, and not recommended. We should avoid adding support for SHA224 unless it's absolutely required.